bower.json

```diff
@@ -2,14 +2,14 @@
   "name": "threesixty-slider",
   "description": "360 degree Image Slider",
   "main": "dist/threesixty.min.js",
-  "version": "2.0.5",
+  "version": "2.0.6",
   "ignore": [
     ".jshintrc",
     ".editorconfig",
     ".bowerrc"
   ],
   "dependencies": {},
   "devDependencies": {
-    "jquery": "~1.8.3"
+    "jquery": "~3.1.1"
```

Comment:
I know that authour is probably not going to update this package, but you don't need to update required version of jQuery, `.on('load', callback)` will work on any jQuery, but to avoid some very old version, will be great to define that this package works with `>=1.8.3`